test/Microsoft.TestPlatform.AcceptanceTests/RunsettingsTests.cs

```diff
@@ -214,12 +211,6 @@ public void RunSettingsWithParallelAndPlatformX64(RunnerInfo runnerInfo)
             var targetPlatform = "x64";
             var testhostProcessName = new[] { "testhost", "dotnet" };
             var expectedProcessCreated = 2;
-            if (!this.IsDesktopRunner())
-            {
-                // this creates dotnet hosted vstest console and 2 testhosts one of which is hosted 
-                // in dotnet, so we have two dotnet + 1 testhost.exe
-                expectedProcessCreated = 3;
```

Comment:
is it ok to remove?

Reply:
Yes, this was a special case for the thing described in the comment. We no longer count processes by observing which process started during run. But rather by counting the number of diagnostic logs for host that were produced during run. This makes it possible to run in parallel, and is less fragile in case where vstest.console and testhost both run under dotnet.exe apphost (which is the special case above where we have 2 dlls, running in 2 dotnet testhost.dll, but also 1 dotnet vstest.console.dll, and we could not distinguish which dotnet.exe is actually hosting testhost, and which is hosting vstest.console, so we counted 3 in total).